data/countries/de/rp/data.yaml

```
@@ -277,9 +277,11 @@ state: Rheinland-Pfalz
 district: Speyer
 urls:
   - type: WEBSITE
-    url: https://gruene-speyer.org/
+    url: https://gruene-speyer.de/
```

The URL after redirect includes the `www.`

```suggestion
    url: https://www.gruene-speyer.de/
```